docs/configuration/builder-examples.md

```diff
@@ -91,6 +91,20 @@ builder:
     options: mode=max,image-manifest=true,oci-mediatypes=true
 ```
 
+## [Building without a Dockerfile locally](#building-without-a-dockerfile-locally)
+
+Your application image can also be built using [cloud native buildpacks](https://buildpacks.io/) instead of using a `Dockerfile` and the default `docker build` process. This example uses Heroku's [ruby](https://github.com/heroku/heroku-buildpack-ruby) and [Procfile](https://github.com/heroku/buildpacks-procfile) buildpack to build your final image. 
```

Comment:
```suggestion
Your application image can also be built using [cloud native buildpacks](https://buildpacks.io/) instead of using a `Dockerfile` and the default `docker build` process. This example uses Heroku's [ruby](https://github.com/heroku/heroku-buildpack-ruby) and [Procfile](https://github.com/heroku/buildpacks-procfile) buildpacks to build your final image. 
```